src/Routing/Loader.php

```diff
@@ -27,9 +27,11 @@ public function load($resource, $type = null) : RouteCollection
             $service->add('ping', "libero.content_api.{$config['name']}.ping");
 
             $service->add('items', "libero.content_api.{$config['name']}.item_list.get")
```

Comment:
`setRequirement()` also seems like a soft link between this and the other bundle, `_format` metadata is passed but without a code dependency